selfservice/strategy/code/code_sender_test.go

```diff
@@ -160,4 +160,56 @@ func TestSender(t *testing.T) {
 		})
 	})
 
+	t.Run("case=should be able to disable invalid email dispatch", func(t *testing.T) {
+		for _, tc := range []struct {
+			flow string
+			send func(t *testing.T)
+		}{
+			{
+				flow: "recovery",
+				send: func(t *testing.T) {
+					s, err := reg.RecoveryStrategies(ctx).Strategy("code")
+					require.NoError(t, err)
+					f, err := recovery.NewFlow(conf, time.Hour, "", u, s, flow.TypeBrowser)
+					require.NoError(t, err)
+
+					require.NoError(t, reg.RecoveryFlowPersister().CreateRecoveryFlow(context.Background(), f))
+
+					err = reg.CodeSender().SendRecoveryCode(context.Background(), f, "email", "not-tracked@ory.sh")
+					require.ErrorIs(t, err, code.ErrUnknownAddress)
+				},
+			},
+			{
+				flow: "verification",
+				send: func(t *testing.T) {
+					s, err := reg.VerificationStrategies(ctx).Strategy("code")
+					require.NoError(t, err)
+					f, err := verification.NewFlow(conf, time.Hour, "", u, s, flow.TypeBrowser)
+					require.NoError(t, err)
+
+					require.NoError(t, reg.VerificationFlowPersister().CreateVerificationFlow(context.Background(), f))
```

Comment:
You can even make `reg.VerificationFlowPersister().CreateVerificationFlow` and `reg.RecoveryFlowPersister().CreateRecoveryFlow` the functions of the test case structs. Then this gets sooo much easier to read.

Reply:
Ah nevermind, I thought it is the same struct.